server/controllers/debian.js

```diff
@@ -45,6 +60,71 @@ module.exports.update_fqdn = (req, res) => {
             }
         });
 
-        res.status(200).send({message: 'I reconfigure your cozy with: ' + params.fqdn});
+        res.status(200).send({ message: 'I reconfigure your cozy with: ' + params.fqdn });
     }
 };
+
+module.exports.host_halt = (req, res) => {
+    // halt the host
+
+    var exec = require('child_process').exec,
+        child;
+
+    const config_filename = '/etc/cozy/self-hosting.json';
+    var halt_script = '/usr/local/sbin/debian-halt.sh';
```

Comment:
sorry I don't understand, I used the same code as the fqdn function...
I changed all the paths to "/usr/local/sbin/", and it is configurable through the file /etc/cozy/self-hosting.json